examples/with-redux/store.js

```diff
@@ -53,8 +53,8 @@ export const initializeStore = (preloadedState) => {
   // with the current state in the store, and create a new store
   if (preloadedState && store) {
     _store = initStore({
-      ...store.getState(),
       ...preloadedState,
+      ...store.getState(),
```

Comment:
This change will prioritize the local state over the state that's coming from the page, which is not ideal.